cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -64,6 +64,8 @@ API Changes
 Known Issues
 ------------
 
+- CDAP has been tested on and supports CDH 4.2.x through CDH 5.3.x. It has not been tested on later versions.
```

Comment:
But maybe what's more germane is to change
```
HBase 0.94.2+, 0.96.0+, 0.98.0+
```
to something such as
```
HBase 0.94.2.x, 0.96.x, 0.98.x
```
Is that correct? Did we stop at 0.98?